packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1090,3 +1102,75 @@ class _ClipsWithShapeBorder extends _MatchRenderObject<RenderClipPath, Null> {
   Description describe(Description description) =>
     description.add('clips with shape: $shape');
 }
+
+class _CoversSameAreaAs extends Matcher {
+  _CoversSameAreaAs(
+    this.expectedPath, {
+      @required this.areaToCompare,
+      this.sampleSize = 20,
+  }) : maxHorizontalNoise = areaToCompare.width / sampleSize,
+       maxVerticalNoise = areaToCompare.height / sampleSize
+  {
+    random = new math.Random(1);
```

Comment:
maybe add a comment that mentions that the seed is intentionally fixed to make the tests deterministic.

Reply:
done